engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/TailCall.java

```diff
@@ -246,6 +257,15 @@ private void collectTailCandidatesExpression(
         Expression expression, java.util.Map<IR, Boolean> tailCandidates) {
       switch (expression) {
         case Function function -> collectTailCandicateFunction(function, tailCandidates);
+        case IfThenElse ite -> {
+          if (isInTailPos) {
+            // Note [Analysing Branches in Case Expressions]
```

Comment:
leftover comment

Reply:
The comment isn't that "off". Updated in f2d5574ca2